week-2/2-mandatory/2-function-creation.js

```diff
@@ -18,11 +18,8 @@ Tip: use logical operators
 function validate(num) {}
 
 /* 
-Write a function that removes an element from an array
-The function must:
-- NOT change the original array
-- return a new array with the item removed
-- remove the item at the specified index
+Write a function that returns an array with the element at the specified index removed.
```

Comment:
Still not clear what the arr param is nor what SHOULD be in the array to be returned.

Is this better?
"write a function that returns a copy of the given array `arr`, but with the element at the given index, `index` removed.  The function must NOT change the original array, `arr`."

Reply:
Good suggestion, thanks! Done!